corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -405,7 +352,7 @@ def create_sync_log(self):
 
     @property
     def restore_class(self):
-        return get_restore_class(self.domain, self.user)
+        return FileRestoreResponse
```

Comment:
Could even change this to be

`restore_class = FileRestoreResponse`

for extra points!